pdr_backend/pdr_dashboard/dash_components/plots.py

```diff
@@ -130,55 +171,104 @@ def _make_figures(fig_tup: Tuple) -> Tuple[go.Figure, go.Figure, go.Figure]:
 
 @enforce_types
 def get_figures_and_metrics(
-    payouts: Optional[List], feeds: ArgFeeds, predictoors: List[str]
-) -> Tuple[go.Figure, go.Figure, go.Figure, float | None, float | None, float | None]:
+    payouts: Optional[List], feeds: ArgFeeds, predictors: List[str]
+) -> FiguresAndMetricsResult:
     """
     Get figures for accuracy, profit, and costs.
     Args:
         payouts (list): List of payouts data.
         feeds (list): List of feeds data.
-        predictoors (list): List of predictoors data.
+        predictors (list): List of predictors data.
     Returns:
-        tuple: Tuple of accuracy, profit, and costs figures, avg accuracy, total profit, avg stake
+        FiguresAndMetricsResult: Tuple of accuracy, profit, and
+        costs figures, avg accuracy, total profit, avg stake
     """
     if not payouts:
-        figures = _make_figures(_empty_trio())
-        return figures[0], figures[1], figures[2], 0.0, 0.0, 0.0
+        accuracy_fig, profit_fig, costs_fig = _make_figures(_empty_trio())
+        return FiguresAndMetricsResult(
+            accuracy_fig, profit_fig, costs_fig, 0.0, 0.0, 0.0
+        )
 
     accuracy_scatters, profit_scatters, stakes_scatters = [], [], []
```

Comment:
same here

Reply:
it would create more code to make figures one by one, I think it is good as like that

Reply:
and here, I mean that empty_trio can be redesigned to return an "empty" FiguresAndMetricsResult. Subsequent assignments to be done on the attributes of that figures_and_metrics.